PIL/_imaging.pyi

```diff
@@ -0,0 +1,104 @@
+if False:
+    from PIL.aliases import *
+
+    from typing import Any  # temporarily, hopefully ;)
+    from typing import Optional, List, Text
+
+PILLOW_VERSION = ...
+DEFAULT_STRATEGY = ...
+FILTERED = ...
+HUFFMAN_ONLY = ...
+RLE = ...
+FIXED = ...
+
+# List of functions is incomplete
+
+def alpha_composite(core1: ImagingCore, core2: ImagingCore) -> ImagingCore: ...
+def blend(core1: ImagingCore, core2: ImagingCore, alpha: float) -> ImagingCore: ...
+def fill(mode: Mode, size: Size, color: Any) -> Any: ...
+def new(mode: Mode, size: Size) -> ImagingCore: ...
+
+def map_buffer(target_data: Any, size: Size, decoder_name: Text,
+               bbox: Optional[Any], offset: int, args: Tuple[Mode, int, int]
+	      ) -> ImagingCore: ...  ### what are data & bbox?
+
+def crc32(buffer: Any, hilo: Tuple[int, int]=(0,0)) -> Tuple[int, int]: ...
+### buffer is bytesarray? Should Tuple[int, int] be another named type?
```

Comment:
Buffer is read only byteslike object. I wouldn't worry about that tuple being another named type, I don't think it's use elsewhere.